docs/operator-manual/metrics.md

```diff
@@ -147,6 +147,13 @@ The Application Set controller exposes the following metrics for application set
 | `argocd_kubectl_request_retries_total`            |  counter  | Number of kubectl request retries.                                                                                                                                                          |
 | `argocd_kubectl_transport_cache_entries`          |   gauge   | Number of kubectl transport cache entries.                                                                                                                                                  |
 | `argocd_kubectl_transport_create_calls_total`     |  counter  | Number of kubectl transport create calls.                                                                                                                                                   |
+| `argocd_github_api_requests_total`     |  counter  | Number of Github API calls. It contains labels for the name and namespace of an applicationset.|
```

Comment:
Format table + add info block that all `argocd_github_api` commands will only be enabled when the flag is configured.